pkg/cmd/roachprod/main.go

```diff
@@ -885,6 +885,28 @@ var getProvidersCmd = &cobra.Command{
 	},
 }
 
+var startGrafanaCmd = &cobra.Command{
+	Use:   `start-grafana <cluster> [<grafanaConfigURL>]`,
+	Short: `spins up a promethius and grafana instance on an each roachprod node.`,
+	Long: `by default, the prom and grafana instances on the lowest numbered node in the cluster
```

Comment:
>  I configure scraping the crdb binary on the last node as well, in case the crdb binary is indeed running on the last node.

Sounds reasonable.

> should these roachprod grafana cmds target the list of nodes returned by c.TargetNodes(), i.e. all nodes the user wants to target in the command, or c.Nodes, all nodes in the roachprod cluster? IMHO, i think it's fine to give the user more control.

Few threads below and above. I think for now avoid TargetNodes() will make your life easier. If folks want more control, we can always make this more complicated in the future.

Reply:
Done.